src/commandHandlers/issue.ts

```diff
@@ -1,6 +1,6 @@
 import axios from 'axios';
 import { Message, MessageEmbed } from 'discord.js';
-import { issueRequestConfig } from '../config';
+import config, { issueRequestConfig } from '../config';
```

Comment:
We should only import what we need
`import { issueRequestConfig, COLORS } from '../config';
What do you think?

Reply:
I actually had the same idea but I think my javascript is rusty, because this threw an error.

I was trying to figure it out all day and couldn't.

Reply:
@AllanRegush importing `COLORS` like that doesn't work because that is a field of the object that is exported by the `config` module. I think importing `import config, { issueRequestConfig } from '../config'` is fine, what do you think?